guides/database-integrating-with-postgresql.adoc

```diff
@@ -0,0 +1,739 @@
+= Integrating with a PostgreSQL database
+:summary: Learn how to configure a datasource to connect a PostgreSQL database
+:includedir: _includes
+include::{includedir}/_attributes.adoc[]
+:prerequisites-time: 20
+:postgre-sql-user: postgres
+:postgre-sql-password: admin
+:postgre-sql-port: 5432
+:postgre-sql-host: localhost
+:postgre-sql-database: bookstore_db
+:postgre-docker-image: docker.io/library/postgres
+:postgre-sql-kubernetes-service-name: postgres-service
+
+In this guide, you will learn how to configure WildFly to connect to a PostgreSQL database. You will create a simple Book Store API application to manage books stored in the database using https://jakarta.ee/specifications/restful-ws/[Jakarta RESTful Web Services (Jakarta REST), window=_blank].
+
+include::{includedir}/_prerequisites.adoc[]
+
+* Docker or any Open Container Initiative engine installed. This guide uses https://podman.io/[Podman, window=_blank].
+
+== Database
+
+=== PostgreSQL
+
+We will use PostgreSQL as the database server in its containerized version: see https://hub.docker.com/_/postgres[PostgreSQL Official Image, window=_blank].
+
+Start PostgreSQL database in a container with:
+
+[source,bash,subs="normal"]
+----
+podman run --rm --name bookstore \
+  -p {postgre-sql-port}:{postgre-sql-port} \
+  -e POSTGRES_PASSWORD={postgre-sql-password} \
+  -e POSTGRES_USER={postgre-sql-user} \
+  -e POSTGRES_DB={postgre-sql-database} \
+  {postgre-docker-image}
+----
+
+NOTE: we started the container with the `--rm` flag so it can be disposed of automatically when we stop it.
+
+
+== Application
+
+=== Create a new Maven project
+
+We are going to use the *WildFly Getting Started Archetype* to create the base structure of our Book Store API application.
```

Comment:
Perhaps make this a link?

Reply:
Done. 

Added https://github.com/wildfly/wildfly-archetypes/tree/main/wildfly-getting-started-archetype link